feature-libs/product-configurator/rulebased/components/overview-bundle-attribute/configurator-overview-bundle-attribute.component.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, Input, OnInit } from '@angular/core';
+import {
+  ImageGroup,
+  Product,
+  ProductScope,
+  ProductService,
+} from '@spartacus/core';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Configurator } from '../../core/model/configurator.model';
+import { ConfiguratorPriceComponentOptions } from '../price/configurator-price.component';
+
+interface ProductExtended extends Product {
```

Comment:
It is also defined in `ConfiguratorAttributeProductCardComponentOptions`.

Reply:
Done

Reply:
ProductExtended has been removed